src/box/lua/config/configdata.lua

```diff
@@ -364,6 +424,18 @@ function methods.missing_names(self)
     return missing_names
 end
 
+-- Cluster configuration.
+--
+-- It is given as is after merging from all the configuration
+-- sources. Default values are NOT applied. Variables are NOT
+-- substituted.
+--
+-- Use :get() to receive an instance config for a particular
+-- instance with applied defaults and substituted variables.
+function methods.cconfig(self)
```

Comment:
Do we really need to introduce this method now, when we have `get` and `instances`? If we do, I'd name it differently. Maybe, `raw_config` to emphasize that the config is returned as is (without applying defaults and substituting variables)?

Still, I'd probably avoid introducing this method now because it looks controversial and isn't really needed AFAIU.

Reply:
It gives me a confidence that if I've missed some usage scenario, a user is able to acquire a raw cluster configuration and traverse it on its own.

However, this feature was not asked explicitly and I've no any usage scenario that wouldn't be covered by `:instances()` + `:get()`.

I agree to hide it, at least for now.

----

I would leave some words regarding possible representations of the cluster configuration.

Substituting variables is not possible for a cluster configuration in a general case. A replicaset level option may have `{{ instance_name }}`. The substitution has sense only in context of a particular instance.

Applying defaults on the whole cluster configuration would break ability to calculate an effective option for a particular instance. If an option is set only on the replicaset level, applying defaults to the instance level would give a higher priority to the default.

OTOH, we can apply defaults only on the global level. I'm not sure that I like it, but I don't see how it may break anything.

----

Regarding the name of the method: I like such concise names. 
`iconfig` and `cconfig` are used internally across the code, they look like a one word. Maybe it is not fair and I should use two words instead of constructing a domain specific pseudo-word. I don't know how to deduce a best general rule.

Maybe it should be `:get(nil, {cluster = true})`.